### PR TITLE
Newly generated 20 year unscheduled downtime file and description in …

### PR DESCRIPTION
**Pull request by cathypetry**
…Sphinx docs.

The original unschedDown.conf file has 165 nights of downtime. I used randomDown.py to generate a single (not medianed) 20 year file, with the same name which has 322 total nights or 161 nights per 10 years of downtime. The mean of the distributions from which the nights are drawn is 159 nights per 10 years. I added some text to doc/source/system.rst to describe this.

### CODE REVIEW

File: doc/source/system.rst [hunk 1]
```diff
@@ -60,6 +60,7 @@ schedDown.conf
 
 unschedDown.conf
 ----------------
+Unscheduled downtime is a single realization of an algorithm which draws random numbers on each day and evaluates whether and how long (integer days) observing stops due to any of four types of events. A minor event, such as a power supply failure, causes a closure for the remainder of the night and next day, and happens 5 out of 365 days. An intermediate event, such as the repair to the filter mechanism, rotator, hexapod or shutter, causes a closure for 3 nights and happens 2 out of 365 days. A major event triggers a closure for 7 nights and occurs 1 out of 730 days. A catastrophic event, such as replacing a raft,  triggers a closure for 14 nights and occurs once in 10 years.
```
Comment (mareuter) [inline doc/source/system.rst]: Small niggly. There is an extra space after the comma in the last sentence.

Comment (mareuter) [inline doc/source/system.rst]: +1 Except for the niggly noted, this looks fine. You can merge the PR when the change is done. No need to recheck.

**Comment by mareuter** [comment]
+1 Except for the niggly noted, this looks fine. You can merge the PR when the change is done. No need to recheck.

**Comment by cathypetry** [comment]
Niggly space after comma removed.

**Pull request merged**